pkg/bpmn_engine/engine_properties.go

```diff
@@ -7,10 +7,9 @@ import (
 )
 
 type ProcessInfo struct {
-	BpmnProcessId string // The ID as defined in the BPMN file
-	Version       int32  // A version of the process, default=1, incremented, when another process with the same ID is loaded
-	ProcessKey    int64  // The engines key for this given process with version
-
```

Comment:
why was this line removed?
I propose leaving the former code unchanged.

Reply:
I am not use github well. I have a mongodb branch. i think this is a merge mistaken